packages/models/src/compare/compare.ts

```diff
@@ -0,0 +1,25 @@
+import { ErrorResponse, SuccessResponse } from "src/index";
+
+export interface AddCompareItem {
+    productId: string;
+}
+
+export interface CompareItems {
+    productId: string;
+    products?: any;
```

Comment:
why any? don't we know the types of products?

Reply:
I will define the type after adding product module.
It might be required to populate product in future.

Reply:
now you can integrate with the product module.